templates/home.html

```diff
@@ -147,13 +185,45 @@ <h5 class="modal-title">Edit Animal</h5>
         });
     }
 
+    // Find animals by their name
+    function findAnimals() {
+        var name = $('#search_by_name').val();
+        if (name) {
+            $.getJSON('/animals', {name: name}, function(data) {
+                $('#animalsList').empty();
+                data.animals.forEach(animal => {
+                    console.log(animal)
```

Comment:
better to remove prints and console.log() from code which can be moved to production

Reply:
@marcustas thanks for the feedback! removed